doc/work_management/jira.rst

```diff
@@ -70,19 +80,29 @@ Task Process
     
         * If task is a fixed length (i.e. training/conferences/etc.) create one task (see task creation above).
         * Fixed length tasks can be larger than 5 Story Points.
+        * In case a non-fixed length task needs to be larger than 5 Story Points, discuss with the Sprint Lead or manager.
+        * Short tasks may be 0.5 Story Point.
 
     * If added during a Sprint in progress, notify Sprint Lead if this is out of scope work for the current Sprint.
-    * if work is to be completed during the current Epic's time frame, link to the Epic.
+    * If work is to be completed during the current Epic's time frame, link to the Epic.
 
         * If not, notify Sprint Lead that a new epic is needed.
 
+    * If less than 10 days are worked during a sprint (sickness, vacation, holidays, etc.), reduce the total number of Story Points proportionally.
+
+        * For each day not worked, subtract 0.8 Story Point.
+        * Round up to the nearest 0.5 Story Point to avoid having insufficient Story Points at the end of the fiscal year.
+
     * Required fields to create a task.
 
         Project:
-            Data Management(DM).
+            Data Management (DM).
```

Comment:
This will most likely change when we go to operations - actually have our own project. - Maybe add this as a note?